src/DssSpell.sol

```diff
@@ -211,6 +245,21 @@ contract DssSpellAction is DssAction {
         DssExecLib.updateCollateralPrice("RWA002-A");
 
         // ---------- Transfer Spark Proxy Admin Controls ----------
+        TransferOwnershipLike(SPARK_TREASURY_CONTROLLER).transferOwnership(SPARK_PROXY);
+        ChangeAdminLike(SPARK_TREASURY).changeAdmin(SPARK_PROXY);
+        ChangeAdminLike(SPARK_TREASURY_DAI).changeAdmin(SPARK_PROXY);
```

Comment:
Can't see in deploy sdcipt

Reply:
It's present in the version that was used for initial deploy. Dai treasury has since been removed as it is no longer needed.